app/src/main/kotlin/com/fernandocejas/sample/core/network/NetworkModule.kt

```diff
@@ -0,0 +1,54 @@
+package com.fernandocejas.sample.core.network
+
+import co.touchlab.kermit.Logger
+import com.fernandocejas.sample.core.di.Feature
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.okhttp.OkHttp
+import io.ktor.client.plugins.HttpTimeout
+import io.ktor.client.plugins.cache.HttpCache
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.plugins.logging.LogLevel
+import io.ktor.client.plugins.logging.Logging
+import io.ktor.http.ContentType
+import io.ktor.serialization.kotlinx.json.json
+import kotlinx.serialization.json.Json
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.module
+import io.ktor.client.plugins.logging.Logger as KtorLogger
+
+fun networkFeature() = object : Feature {
```

Comment:
Since this might not become a feature, the DI part should be rethought then. If do not see any issues to continue having a DI core module that includes all the cross cutting dependencies.